handbook/ops/bizops/user_definitions.md

```diff
@@ -1,37 +1,67 @@
 # User metrics definitions
 
-## On-premise
+We use metrics to guide prioritization and planning. By defining metrics against the kinds of behaviors that we want to drive, we can plan features and measure results against the outcomes we're looking for. The three most important metrics we track are acquisition (of visitors), activation (turning them into active users), and retention (keeping them month to month).
 
-### Metrics definitions
+Specific targets for FY22 can be found on the [product home page](../../product/index.md#goals).
 
-**On premise**: a user is considered 'active' when they are signed-in and trigger any event over the specified time period. This could be anything from a page view inside the product to a hover using one of Sourcegraph's browser extensions to a code monitoring email notification.
+## User states
 
-Time periods we track:
-
-- Daily active user (DAU)
-- Weekly active user (WAU)
-- Monthly active user (MAU)
-
-### Growth metrics user categories
-
-We track the following categories in pings for each month. The explanations are framed as though we're talking about the current month, but apply to any given month.
+Users can be in a number of different states. We track the following categories in pings for each month. The descriptions below are framed as though we're talking about MAU, but the same definitions apply to whatever period is being considered.
```

Comment:
I would remove the words "in pings" from the following sentence:" We track the following categories in pings for each month."

Reply:
```suggestion
Users can be in a number of different states. We track the following categories for each month. The descriptions below are framed as though we're talking about MAU, but the same definitions apply to whatever period is being considered.
```